staging/src/k8s.io/client-go/util/certificate/certificate_manager.go

```diff
@@ -448,6 +448,7 @@ func (m *manager) run() {
 
 		// Don't enter rotateCerts and trigger backoff if we don't even have a template to request yet
 		if m.getTemplate() == nil {
+			logger.V(2).Info("No certificate template available")
```

Comment:
These are debug style logs, so they need to be at least log level 4.

Reply:
agree, let's move all of them to level 4

Reply:
Makes sense, I'll change to level 4